monitoring/monitors/viya/podMonitor-sas-go-pods.yaml

```diff
@@ -22,11 +22,32 @@ spec:
     #   regex: golang
     #   action: keep
     - sourceLabels: [__meta_kubernetes_pod_label_app_kubernetes_io_name]
-      regex: "sas-arke|sas-launcher"
+      regex: "sas-arke|sas-launcher|sas-redis-operator"
       action: drop
     - sourceLabels: [__meta_kubernetes_pod_annotation_sas_com_kustomize_base]
       regex: "golang"
       action: keep
+    - action: replace
```

Comment:
I don't know if it matters or not, but the order of the filter is "not like the others."
I'm not sure if this is a big deal or not, but I can go back and update them for consistency.

Reply:
Yeah, I'm not crazy about having inconsistent ordering of things given how tough it often is to keep track of things in yaml files.  I think the issue is complicated  here by the fact that we were using sourceLabels as the "first" item in every set and these new relabelings include multiple sourceLabel values instead of just one.  I think tragetLabel might be a better "id"/first row since (I think) there is always only one target.  But I'm not sure if we need/want to reorder everything.